code/items/ammo/MagnumAmmo.cs

```diff
@@ -7,6 +7,8 @@ namespace TTT;
 public partial class MagnumAmmo : Ammo
 {
 	public override AmmoType Type => AmmoType.Magnum;
-	public override int DefaultAmmoCount => 5;
+	public override int DefaultAmmoCount => 6;
+	public override int MaxPlayerAmmo => 12;
```

Comment:
The ammo box itself doesn't need to know about the max player ammo. It should probably be something like
```csharp
public int[] WeaponsOfAmmoType = new int[] { 0, 0, 0, 0, 0, 0 };
```
found in Inventory.cs. I think it makes sense to store the max ammo amount on the player, @kolexxx can suggest something better.